src/main/java/seedu/eylah/expensesplitter/model/person/Person.java

```diff
@@ -79,10 +79,11 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
-                .append(" Amount: ")
-                .append(getAmount())
-                .append(" Tags: ");
+        String formattedName = getName().toString().substring(0, 1).toUpperCase()
+                + getName().toString().substring(1).toLowerCase();
+        builder.append(formattedName)
+                .append(" owes: $")
+                .append(getAmount());
```

Comment:
I think this is a good modify for the output to the user which could increase the readability of the result.